packages/pages/src/vite-plugin/plugin.ts

```diff
@@ -5,6 +5,7 @@ import {
   ProjectStructureConfig,
 } from "../common/src/project/structure.js";
 import { build } from "./build/build.js";
+import _ from "lodash";
```

Comment:
nit but probably better just to import the one method instead of all of lodash `{ merge } from "lodash"`

Reply:
there's an error if we do named export like that:
<img width="796" alt="Screen Shot 2022-10-26 at 12 18 13 PM" src="https://user-images.githubusercontent.com/36055303/198080127-cf511414-9eb7-47e1-9bff-9451b5a19924.png">
I will keep it as is for now

Reply:
oof
does `import merge from "lodash/merge"` work? if not that's fine

Reply:
`import merge from "lodash/merge.js"` works, will update